### PR TITLE
Update README 📝 

### PR DESCRIPTION
**Pull request by huymq1710**
### What is the purpose of this pull request?
To make updates to the README file, aiming to improve its clarity and cleanliness. 

The intention is to enhance the readability and understanding of the information provided in the README

### When do you want to merge?
ASAP 

This is my first PR, so if there is anything else I should do, please let me know. Hope to contribute more for Youngmonkeys.

### CODE REVIEW

**Comment by huymq1710** [comment]
@tvd12 , please take a look :bow:

**Comment by tvd12** [comment]
@boushphong pls review

**Review by boushphong — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ Ezyfox Server's benchmark involved broadcasting messages to 1000 concurrent user
 
 # Introduction
 
+A Free and Open Solution for Multiplayer Game Development
```
Comment (vu-luong) [inline README.md:37]: I don't think ezyfox-server is just for game dev. We can build any real-time stuffs with it.

File: README.md [hunk 2]
```diff
@@ -34,11 +34,13 @@ Ezyfox Server's benchmark involved broadcasting messages to 1000 concurrent user
...
+
 To develop online games rapidly, developers often need to use a game server engine like SmartFoxServer or Photon. Unfortunately, these engines come with a high price tag, especially as more and more users engage with our products. That's why we developed the EzyFox ecosystem, which aims to be free and open for anyone interested in building multiplayer games and applications.
 
 The EzyFox ecosystem supports a wide range of essential components for enterprise product development, including TCP, UDP, WebSocket protocols with SSL encryption, HTTP RESTful API, Remote Procedure Call (RPC) protocol, Database Interaction, Memory Caching, and Message Queue.
 
-With the EzyFox ecosystem, we can avoid the headache of choosing which technologies to use to manage and scale up an application, allowing us to focus solely on implementing business logic.
+With the EzyFox ecosystem, developers can foster creativity, collaboration, and innovation in multiplayer game development, irrespective of their budget constraints. By providing a comprehensive, free, and open solution, we aim to empower developers to bring their gaming visions to life and create memorable experiences for players worldwide.
```
Comment (vu-luong) [inline README.md:43]: I would prefer the original sentence. It seems to stick to the point better. The new sentence has quite general meaning, and doesn't differentiate our framework from others.

File: README.md [hunk 3]
```diff
@@ -160,10 +162,16 @@ mvn test
 - Ask us on [stackask.com](https://stackask.com)
 - Email us at [Dzung](mailto:itprono3@gmail.com)
 
-# Help us by making a donation.
+# Support Us: Make a Meaningful Donation.
```
Comment (vu-luong) [inline README.md:165]: Seems like we're not being case consistent. E.g.,
- Latest version (lower case 'v')
- Tutorial: (redundant ':')

Could you revise all other headings in this MR as well? Thanks

File: README.md [hunk 4]
```diff
@@ -160,10 +162,16 @@ mvn test
...
 
 Currently, our operating budget is fully supported by our own salaries, and all product development is still based on voluntary contributions from a few organization members. The low budget is causing significant difficulties for us. Therefore, with a clear roadmap and an ambitious goal to provide intellectual products for the community, we would greatly appreciate your support in the form of a donation to help us take further steps. Thank you in advance for your meaningful contributions!
 
+Your donation will enable us to:
+- Expand Development
+- Enhance Support
+- Community Growth
+- Promote Openness
```
Comment (vu-luong) [inline README.md:173]: Think these points are not headings, so I would prefer NOT using Title case here

**Review by vu-luong — CHANGES_REQUESTED** [review]

**Review by tvd12 — CHANGES_REQUESTED** [review]
@huymq1710 PLTAL

File: README.md (hunk 3 above)
Comment (huymq1710) [reply README.md]: Fix all headings to be case consistent

**Review by vu-luong — APPROVED** [review]
LGTM

**Review by boushphong — COMMENTED** [review]
LGTM now!

File: README.md [hunk 5]
```diff
@@ -34,6 +34,8 @@ Ezyfox Server's benchmark involved broadcasting messages to 1000 concurrent user
 
 # Introduction
 
+An Open Solution for all your Real-Time needs
```
Comment (tvd12) [inline README.md:37]: `An Open Solution for all your Real-Time needs.`

**Review by tvd12 — CHANGES_REQUESTED** [review]
@huymq1710 PLTAL

**Review by tvd12 — APPROVED** [review]
LGTM

**Pull request merged**